bootstrap/021_external_functions.sql

```diff
@@ -109,6 +109,17 @@ EXCEPTION
         RAISE;
 END;
 
+create or replace function internal.wrapper_ef_run(tenant_info object, request object)
```

Comment:
`tenant_info` appears to be unused by `ef_run`. Do we need to propagate that into this wrapper?

Instead, just make these calls `internal.ef_run(NULL, request)`

Reply:
fixed it